administrator/components/com_admin/sql/updates/mysql/5.0.0-2023-09-04.sql

```diff
@@ -0,0 +1,6 @@
+-- Reset default style in administrator
+UPDATE `#__template_styles` SET `home` = 0 WHERE `client_id` = '1';
```

Comment:
```suggestion
UPDATE `#__template_styles` SET `home` = '0' WHERE `client_id` = '1';
```
It's a char7 and you quoted it in the statement below, too.

Reply:
perfect then I mixed up both^^ is psql correct at least?

Reply:
Yes, PG was OK.